### PR TITLE
Update the Google.Devtools.AspNet project.json

### PR DESCRIPTION
**Pull request by iantalarico**
Update the Google.Devtools.AspNet project.json descriptions to not be exclusive to error reporting.

### CODE REVIEW

File: apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/project.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Unit tests for the Google Stackdriver Error Reporting ExceptionLogger library.",
+  "description": "Unit tests for the Google Stackdriver Insturmnetation Libraries",
```
Comment (jskeet) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/project.json]: Nit: "Instrumentation" here and before.

**Comment by jskeet** [comment]
Hopefully rebasing this PR will radically reduce the number of commits...

**Comment by iantalarico** [comment]
I just did a rebase, I'm thinking I did something wrong...

**Comment by jskeet** [comment]
It may be worth starting a new branch given the size of the change you *want* (a small number of lines in one file) vs what it's looking like now (243 files changed)!

**Pull request closed without merging**